### PR TITLE
CHROMEOS test-configs.yaml: Update chromeos-flash paths

### PR DESCRIPTION
**Pull request by mgalka**
Update kernel, modules, initrd and rootfs paths for chromeos-flash job.

### CODE REVIEW

**Comment by mgalka** [comment]
chromeos-flash job tested with updated URLs
https://lava.collabora.co.uk/scheduler/job/6073007

File: config/core/test-configs.yaml [hunk 1]
```diff
@@ -164,10 +164,10 @@ test_plans:
       job_timeout: '60'
       docker_image: 'kernelci/chromeos-flash'
       chromeos_image: 'chromiumos_test_image-R94-14150.19.0-octopus.bin'
-      flashing_kernel: https://people.collabora.com/~mgalka/stable/linux-5.10.y/v5.10.95/x86_64/x86_64_defconfig+x86-chromebook/gcc-10/kernel/bzImage
-      flashing_modules: https://people.collabora.com/~mgalka/stable/linux-5.10.y/v5.10.95/x86_64/x86_64_defconfig+x86-chromebook/gcc-10/modules.tar.xz
-      flashing_rootfs: http://storage.chromeos.kernelci.org/images/rootfs/chromeos/20220328.0/bullseye-chromeos-flash/amd64/full.rootfs.tar.xz
-      flashing_ramdisk: http://storage.kernelci.org/images/rootfs/debian/buster/20210503.0/amd64/initrd.cpio.gz
+      flashing_kernel: http://storage.chromeos.kernelci.org/images/kernel/v5.17.2-x86_64%2Bx86_chromebook/kernel/bzImage
+      flashing_modules: http://storage.chromeos.kernelci.org/images/kernel/v5.17.2-x86_64%2Bx86_chromebook/modules.tar.xz
```
Comment (gctucker) [inline config/core/test-configs.yaml:168]: How was `%2B` added to the URL? Maybe we could just use `+`?

**Review by gctucker — COMMENTED** [review]
LGTM - just a minor thing I'm not sure of.

**Comment by mgalka** [comment]
Re-tested with `%2B` changed to `+`
https://lava.collabora.co.uk/scheduler/job/6076678

**Review by gctucker — APPROVED** [review]

**Pull request merged**